Libraries/MiscDrivers/Display/tft_ssd2119.c

```diff
@@ -926,10 +927,10 @@ int MXC_TFT_Init(void)
     memset(&images_header, 0, sizeof(Header_images_t));
 
     // Is there any image data to work with?
-    if (_bin_start_ != _bin_end_) {
+    if ((unsigned int *)&_bin_start_ != (unsigned int *)&_bin_end_) {
         images_start_addr = (unsigned char *)&_bin_start_;
         // set header
-        memcpy(&images_header, images_start_addr, sizeof(Header_images_t));
+        images_header = *((Header_images_t *)&_bin_start_);
```

Comment:
Why is this cast needed?

Reply:
GCC 13.2 wants to be informed about the memory address is enough for the related memcopy. We're giving a struct type pointer which we claim it will not overflow with the memory copying. Actually *(&_bin_start_) is enough but I added (Header_images_t *) casting for readability.

Reply:
This does not add to readability, it reduces readability and adds the possibility of mis-aligned access, and access to unallocated memories, and incompatible data structures.
This should be
images_header = _bin_start_;